easybuild/easyblocks/g/gcc.py

```diff
@@ -354,10 +394,15 @@ def configure_step(self):
         if self.cfg['languages']:
             self.configopts += " --enable-languages=%s" % ','.join(self.cfg['languages'])
 
-        if self.cfg['withnvptx']:
-            if self.iter_idx == 0:
+        if self.cfg['withnvptx'] or self.cfg['withamdgcn']:
+            if self.build_stages[self.iter_idx] == HOST_COMPILER:
```

Comment:
Would it make sense to introduce a `self.current_stage`, and set that once (in `prepare_step`?) to `self.build_stages[self.iter_idx]`? That would make this easier to understand, I think:

```python
if self.current_stage == HOST_COMPILER: